saws-testing/src/test/scala/com/ambiata/saws/testing/S3Spec.scala

```diff
@@ -0,0 +1,13 @@
+package com.ambiata.saws.testing
+
+import org.specs2.matcher.Parameters
+import org.specs2.ScalaCheck
+import org.specs2.Specification
+import org.specs2.specification.Fragments
+
+abstract class S3Spec extends Specification with ScalaCheck {
+  override def map(fs: =>Fragments) = section("aws") ^ super.map(fs)
+
+  override implicit def defaultParameters: Parameters =
+    new Parameters(minTestsOk = 5, workers = 3)
```

Comment:
I am a bit worried about this, how long do you think before someone extends this not realizing or copy-pasting from another test, could maybe make the min tests and argument, so you go like `extends S3Spec(5)` or similar in test case so we can at least see in reviews.